apps/user-portal/src/middlewares/consent-management.ts

```diff
@@ -159,23 +165,29 @@ const handleFetchConsentReceipt = ({ dispatch }) => (next) => (action) => {
         return;
     }
 
-    const id: string = action.payload;
-    const requestConfig: HttpRequestConfig = {
-        dispatcher: FETCH_CONSENT_RECEIPT,
-        headers: {
-            "Accept": "application/json",
-            "Access-Control-Allow-Origin": CLIENT_HOST,
-            "Authorization": `Bearer ${ AuthenticateSessionUtil.getAccessToken(CLIENT_ID, CLIENT_HOST) }`,
-            "Content-Type": "application/json"
-        },
-        method: "get",
-        onError: FETCH_CONSENT_RECEIPT_ERROR,
-        onSuccess: FETCH_CONSENT_RECEIPT_SUCCESS,
-        url: ServiceResourcesEndpoint.receipts + `/${ id }`
-    };
-
-    // Dispatch an API request action.
-    dispatch(apiRequest(requestConfig));
+    AuthenticateSessionUtil.getAccessToken()
+        .then((token) => {
+            const id: string = action.payload;
+            const requestConfig: HttpRequestConfig = {
+                dispatcher: FETCH_CONSENT_RECEIPT,
+                headers: {
+                    "Accept": "application/json",
+                    "Access-Control-Allow-Origin": CLIENT_HOST,
+                    "Authorization": `Bearer ${ token }`,
+                    "Content-Type": "application/json"
+                },
+                method: "get",
```

Comment:
Keep the method on consistent case. Lowercase or Uppercase everywhere

Reply:
Method types were made uppercase.